manpages-md-extra/bootc-install-config.md

```diff
@@ -18,9 +18,10 @@ that can be overridden in a derived container image.
 
 This is the only defined toplevel table.
 
-The `install`` section supports two subfields:
+The `install`` section supports the following subfields:
 
 - `filesystem`: See below.
+- `disable_composefs`: A boolean, which will use the legacy ostree mode.
```

Comment:
I think this needs more explanation, and maybe even a better name. Where exactly is being `composefs` disabled? In the kernel? In the filesystem layout in the root mount namespace? Might be also worth explaining what's the legacy ostree mode.